Android/Shared/ITMAppConfig.gradle

```diff
@@ -89,15 +89,16 @@ ext {
 }
 
 task genITMAppConfig {
-    def destDir = Paths.get("${project.rootDir}/app/src/main/assets/ITMApplication")
-    if (!Files.exists(destDir))
-        Files.createDirectories(destDir)
-    def itmAppConfig = new File("${destDir.toString()}/ITMAppConfig.json")
-    if (!itmAppConfig.exists()) {
-        itmAppConfig.createNewFile()
+    dependsOn ":itmsamplesshared:copyITMAssets"
```

Comment:
@tcobbs-bentley This task doesn't really depend on copyITMAssets, it can be independent I think. We would then have preBuild depend on both of these tasks instead.

Reply:
Are you asking me if it depends on copyITMAssets? If so, the answer is that I'm not sure. It may depend on that task to create the directory that it puts the generated ITMAppConfig.json file into. Then again, it might not.

Reply:
I've removed the dependency from genITMAppConfig.